app/src/main/res/values-fr/strings.xml

```diff
@@ -59,4 +59,21 @@
 
     <string name="sign_in_with_google">Se connecter avec Google</string>
     <string name="welcome_screen_title">Bienvenue à</string>
+    <string name="my_events_screen_name">Mes Événements</string>
+    <string name="my_events_screen_created_event_list_name">Les Événements Créés</string>
+    <string name="my_events_screen_joined_event_list_name">Les Événements Rejoint</string>
+    <string name="my_events_screen_empty_create_event_list">Vous avez créé aucun événement</string>
+    <string name="my_events_screen_empty_join_event_list">Vous avez rejoint aucun événement</string>
+    <string name="date_tools_from">De</string>
+    <string name="date_tools_until">Jusqu\'au</string>
+    <string name="date_tools_at">à</string>
+    <string name="event_details_screen_number_of_guests">Singes ont rejoint</string>

```

Comment:
The French translation isn't coherent with the English one, do you consider French people are monkeys and English one are not ????

Reply:
I had believe we all agreed on the fact that guests were called "monkeys" in the app to go with the theme, then I received a review that I needed to change it to guest and I did but only did the English version. The French one went over my head. Thanks!